### ISSUE TITLE
"go to declaration" doesn't work across multiple windows

### ISSUE THREAD

**Issue by cprice404** [cursiveclojure/cursive#740]
I use a tiling window manager, and almost always have multiple discrete IDEA windows open for a given project (e.g. by dragging the tab for an open file out to outside of the main window, so that it floats as a separate window).

When working in this mode, if I do a ctrl-click or 'go to declaration' for a symbol that is defined in a file that I already have open in a different window, I'd ideally like it to focus that window and jump to the source there.  Instead, it opens up a second copy of the file in the current window... which I end up closing and then navigating to the other file by hand and just doing a 'find' for the string to get to the declaration.

I have no idea whether this is a problem with, well, IDEA, or something cursive-specific.

**Comment by cursiveclojure** [comment]
Yeah, I'm finding this a lot recently. In fact, you don't need two windows - I have a single window but I split it into two panes, and it happens there too. It's definitely happening more recently, maybe it's something changed in v14? I'm not sure. It's definitely an IntelliJ thing either way. I dimly recall reading an issue about this a long time ago, I'll see if I can dig it up.
